ci/tests/fputil.bash

```diff
@@ -81,7 +82,7 @@ function integration_test {
         echo "[INFO] Allowing GDS ${SLEEP_TIME} seconds to start"
         sleep ${SLEEP_TIME}
         # Check the above started successfully
-        ps -p ${GDS_PID} 2> /dev/null 1> /dev/null || fail_and_stop "Failed to run GDS layer headlessly"
```

Comment:
This redirect should be kept, as the error on the right is sufficient.

Reply:
Incorporated.